packages/rum-core/src/common/utils.js

```diff
@@ -431,6 +431,11 @@ function isBeaconInspectionEnabled() {
   return false
 }
 
+// redirect info is only available for same-origin redirects
+function isRedirectInfoAvailable(timing) {
+  return timing.redirectStart > 0
```

Comment:
Are there cases when this data can be buggy? Like case with fetchStart before redirectStart or being in unix vs zero based offset timestamps. Check the buggy navigation data test cases to see if its relevant. Not a blocker though, just want to call out.

Reply:
thanks for highlighting this.

I've not found anything that might affect the agent, the only bug I found is [this](https://bugs.chromium.org/p/chromium/issues/detail?id=805831), but doesn't seem relevant for us and nobody has reproduced it after Chromium 30 🧐